updated licensing docs for 8.3 (#41696)
grafana/grafana: docs/sources/enterprise/license/license-restrictions.md

weight = 300
+++

# Understanding Grafana Enterprise licensing

> When you become a  Grafana Enterprise customer, you receive a license that governs your use of Grafana Enterprise.

Reviewer: ```suggestion
As a Grafana Enterprise customer, you receive a license that governs your use of Grafana Enterprise.
```
